pages/donate/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Center, Container, Flex, Link, Text } from '@chakra-ui/react';
+import NextLink from 'next/link';
+
+const Donate: React.FC = () => {
+  return (
+    <>
+      <Flex gap={'150px'} flexDir={'column'}>
+        <Container maxW={'container.lg'} mb="100px">
+          <Flex gap={'60px'} flexDir="column">
+            <div data-cy="donate-top-message" id={'donate-top-message'}>
+              <Flex flexDir="column" gap={'15px'}>
+                <Center>
+                  <Text fontSize="5xl">Any amount helps us reach our goal!</Text>
+                </Center>
+              </Flex>
+            </div>
+            <div data-cy="donate-top-submessage" id={'donate-top-submessage'}>
+              <Flex flexDir="column" gap={'15px'}>
+                <Center>
```

Comment:
This also needs to have a `text-align: center` somewhere. Center does not align text, only elements. If you look at the mobile view you can see how it makes a big difference.